### PR TITLE
adding main layout

### PR DESCRIPTION
**Pull request by benjaglobant**
adding main layout of the calculator

### CODE REVIEW

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -7,12 +7,194 @@
...
+        app:layout_constraintStart_toEndOf="@+id/btn_plus"
+        app:layout_constraintTop_toTopOf="@+id/btn_plus" />
+
+    <Button
+        android:id="@+id/btn_plus"
+        android:layout_width="@dimen/btn_layout_midth"
+        android:layout_height="@dimen/btn_layout_heigth"
+        android:layout_marginStart="@dimen/btn_plus_marginstart"
+        android:layout_marginLeft="@dimen/btn_plus_marginstart"
+        android:layout_marginTop="44dp"
```
Comment (BQuinteros) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: pls don't use hardcode, use @dimen/...

Comment (benjaglobant) [reply OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: Done!

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -7,12 +7,194 @@
...
+
+    <Button
+        android:id="@+id/btn_7"
+        android:layout_width="@dimen/btn_layout_midth"
+        android:layout_height="@dimen/btn_layout_heigth"
+        android:text="@string/num_7"
+        android:textSize="@dimen/btn_textsize"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.192"
```
Comment (nicolbidone) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: pls check if the bias is necessary

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -7,12 +7,194 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/btn_plus"
+        app:layout_constraintVertical_bias="0.264" />
```
Comment (nicolbidone) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: pls check if the bias is necessary

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -7,12 +7,194 @@
...
+    <Button
+        android:id="@+id/btn_equal"
+        android:layout_width="@dimen/btn_layout_midth"
+        android:layout_height="@dimen/btn_layout_heigth"
+        android:text="@string/equal"
+        android:textSize="@dimen/btn_textsize"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/btn_point"
+        app:layout_constraintTop_toTopOf="@+id/btn_0"
+        app:layout_constraintVertical_bias="0.0" />
```
Comment (nicolbidone) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: pls check if the bias is necessary

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -7,12 +7,194 @@
...
+
+    <Button
+        android:id="@+id/btn_6"
+        android:layout_width="@dimen/btn_layout_midth"
+        android:layout_height="@dimen/btn_layout_heigth"
+        android:text="@string/num_6"
+        android:textSize="@dimen/btn_textsize"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
```
Comment (nicolbidone) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: pls check if the bias is necessary

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -7,12 +7,194 @@
...
+    <Button
+        android:id="@+id/btn_0"
+        android:layout_width="@dimen/btn_layout_midth"
+        android:layout_height="@dimen/btn_layout_heigth"
+        android:text="@string/num_0"
+        android:textSize="@dimen/btn_textsize"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="@+id/btn_1"
+        app:layout_constraintTop_toBottomOf="@+id/btn_1"
+        app:layout_constraintVertical_bias="0.0" />
```
Comment (nicolbidone) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: pls check if the bias is necessary

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -7,12 +7,194 @@
...
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        android:layout_marginTop="@dimen/btn_clear_margintop"
+        android:text="@string/clear"
+        android:textSize="@dimen/btn_textsize"
+        app:layout_constraintBottom_toTopOf="@+id/btn_8"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/btn_sub"
+        app:layout_constraintVertical_bias="0.266" />
```
Comment (nicolbidone) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: pls check if the bias is necessary

**Review by nicolbidone — CHANGES_REQUESTED** [review]

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -7,12 +7,194 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/visorPrint"
+        android:layout_width="@dimen/visor_layour_width"
+        android:layout_height="@dimen/visor_layout_height"
+        android:text=""
```
Comment (nicolbidone) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: this is considered hardcode pls instead use @string/empty_string with "" as value

**Review by nicolbidone — CHANGES_REQUESTED** [review]

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (benjaglobant) [reply OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: Done! and thanks by the explanation!

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (benjaglobant) [reply OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: Done! and thanks for the explanation!

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (benjaglobant) [reply OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: Done! and thanks for the explanation!

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml (hunk 5 above)
Comment (benjaglobant) [reply OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: Done! and thanks for the explanation!

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml (hunk 6 above)
Comment (benjaglobant) [reply OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: Done! and thanks for the explanation!

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml (hunk 7 above)
Comment (benjaglobant) [reply OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: Done! and thanks for the explanation!

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml (hunk 8 above)
Comment (benjaglobant) [reply OnBoardingCalculator/app/src/main/res/layout/activity_main.xml]: Done! and thanks for the explanation!

File: OnBoardingCalculator/app/src/main/res/layout/activity_main.xml [hunk 9]
```diff
@@ -4,15 +4,167 @@
...
+        app:layout_constraintStart_toEndOf="@+id/btn_4"
+        app:layout_constraintTop_toTopOf="@+id/btn_4" />
+
+    <Button
+        android:id="@+id/btn_7"
+        style="@style/btn_num_style"
+        android:layout_marginTop="@dimen/btn_seven_margintop"
+        android:text="@string/num_7"
+        app:layout_constraintEnd_toStartOf="@id/btn_8"
+        app:layout_constraintHorizontal_chainStyle="packed"
```
Comment (mauroLonghi) [inline OnBoardingCalculator/app/src/main/res/layout/activity_main.xml:51]: Excellent use of the properties of Contraint Layout!

**Review by BQuinteros — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Pull request merged**